radicle/src/node.rs

```diff
@@ -449,8 +466,22 @@ impl Handle for Node {
         matches!(result, CommandResult::Okay { .. })
     }
 
-    fn connect(&mut self, _node: NodeId, _addr: Address) -> Result<(), Error> {
-        todo!()
+    fn connect(&mut self, nid: NodeId, addr: Address) -> Result<(), Error> {
+        let result = self
+            .call::<_, CommandResult>(CommandName::Connect, [nid.to_human(), addr.to_string()])?
+            .next()
+            .ok_or(Error::EmptyResponse {
+                cmd: CommandName::Connect,
+            })??;
+        match result {
+            CommandResult::Okay { .. } => {
+                log::info!(target: "node", "Successfully connected to {} {}", nid, addr)
+            }
+            CommandResult::Error { reason } => {
+                log::warn!(target: "node", "Failed to connect to {} {}: {}", nid, addr, reason)
+            }
+        }
```

Comment:
I'd remove these log statements because actually the connection will happen asynchronously, and could take some time to establish. Right now unfortunately the command is not designed to wait for the connection to be established, though that's something we should implement at some point.